keras/backend/tensorflow_backend.py

```diff
@@ -395,6 +413,18 @@ def is_keras_tensor(x):
     return hasattr(x, '_keras_history')
 
 
+def is_native_tensor(x):
+    """Returns whether `x` is a native tensor.
+
+        # Arguments
+            x: a potential tensor.
+
+        # Returns
+            A boolean: whether the argument is a native tensor.
+    """
+    return isinstance(x, tf.Tensor)
```

Comment:
`is_keras_tensor()` doesn't work. This does. `tf.train.shuffle_batch` returns type `tf.Tensor`.